### PR TITLE
feat(cli): add explorer

### PR DESCRIPTION
**Pull request by idea404**
add explorer to cli devnet

task: https://app.asana.com/0/1206208509925075/1207138429056645/f

### CODE REVIEW

**Review by pavelnikolov — APPROVED** [review]

**Review by pavelnikolov — APPROVED** [review]

**Review by corverroos — CHANGES_REQUESTED** [review]
explorer-ui is not a go program... Not sure we can use goreleaser to build it. Does this actually work?

File: .github/workflows/release-official.yml [hunk 1]
```diff
@@ -32,4 +32,7 @@ jobs:
         with:
           context: "{{defaultContext}}:explorer/ui"
           push: true
-          tags: omniops/explorer-ui:main,omniops/explorer-ui:${{ steps.git_ref.outputs.short_sha }}
+          tags: |
+            omniops/explorer-ui:main
+            omniops/explorer-ui:${{ steps.git_ref.outputs.short_sha }}
+            omniops/explorer-ui:${GITHUB_REF##*/}  # This adds the tag (e.g., v0.1.8)
```
Comment (pavelnikolov) [inline .github/workflows/release-official.yml:38]: Does this work with the docker/build-push-action? I thought that only `,` sperator works 🤔

**Comment by idea404** [comment]
This PR will require updating the `clicommand.yml` to include the Explorer UI Docker images if https://github.com/omni-network/omni/pull/1151 is merged first

File: cli/cmd/devnet.go [hunk 2]
```diff
@@ -171,6 +171,7 @@ func devnetDefinition(ctx context.Context) (app.Definition, error) {
 
 	defCfg := app.DefaultDefinitionConfig(ctx)
 	defCfg.ManifestFile = manifestFile
+	defCfg.GraphQLURL = "http://explorer_graphql:8080/query"
```
Comment (corverroos) [inline cli/cmd/devnet.go:174]: rather add this as the "default" GraphQLURL [here](https://github.com/omni-network/omni/blob/main/e2e/app/definition.go#L59).

**Pull request merged**